src/pykiso/lib/connectors/cc_pcan_can.py

```diff
@@ -135,25 +144,23 @@ def __init__(
         self.bus = None
         self.logging_activated = logging_activated
         self.raw_pcan_interface = None
-        self.logging_path = None
         # Set a timeout to send the signal to the GIL to change thread.
         # In case of a multi-threading system, all tasks will be called one after the other.
         self.timeout = 1e-6
-        """
-        Extract the base logging directory from the logging module, so we can
-        create our logging folder in the correct place.
-        logging_path will be set to the parent directory of the logfile which
-        is set in the logging module + /raw/PCAN
-        If an AttributeError occurs, the logging module is not set to log into
-        a file.
-        In this case we set the logging_path to None and will just log into a
-        generic logfile in the current working directory, which will be
-        overwritten every time, a log is initiated.
-        """
-        if self.trace_path.is_file():
+        self.trc_count = 0
+
+        # Handle trace path and name
+        if self.trace_path.suffix == ".trc":
```

Comment:
maybe create a method who encapsulate the trace initialization

Reply:
Done